ext/sst/INSTALL.md

```diff
@@ -98,6 +98,13 @@ According to the OS that you're using, you need to rename the `Makefile.xxx` to
 cp Makefile.xxx Makefile    # linux or mac
 make -j4
 ```
+If you are compiling this on Mac, then you'd need to export `DYLD_LIBRARY_PATH`
+```sh
+# go to the base gem5 directory
+cd ../..
+export DYLD_LIBRARY_PATH=:`pwd`/build/RISCV/
```

Comment:
Why specifically `build/RISCV/`?

This would be better if it was generic. E.g., `export DYLD_LIBRARY_PATH=:$GEM5_BASE/<build directory>` 

This being said, this change has *nothing* to do with "added functional accesses to gem5/SST bridge" it should be in a different change.

Reply:
we are supporting the RISCV version as the default one. We can definitely do this but we'd also need to change the README and the INSTALL files to make it more generic. I agree on making it more generic as the default examples we have are `example.py` -> RISCV/default and `arm_example.py` -> ARM.